tutorials/quantum_simulation/Shadow estimation for VQS.ipynb

```diff
@@ -1,20 +1,5 @@
 {
```

Comment:
<p>same as above with the <code>ket</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>